doc/examples/fiber_to_bundle_coherence.py

```diff
@@ -0,0 +1,281 @@
+# -*- coding: utf-8 -*-
+"""
+==============================================
+Fiber to bundle coherence measures
+==============================================
+
+This demo presents the fiber to bundle coherence (FBC) quantitative 
+measure of the alignment of each fiber with the surrounding fiber bundles
+[Meesters2016_HBM]_. These measures are useful in “cleaning” the results of 
+tractography algorithms, since low FBCs indicate which fibers are isolated and 
+poorly aligned with their neighbors, see Fig. 1.
+
+.. figure:: fbc_illustration.png
+   :align: center
+
+   On the left this figure illustrates (in 2D) the contribution of two fiber 
+   points to the kernel density estimator. The kernel density estimator is the 
+   sum over all such locally aligned kernels. The local fiber to bundle coherence
+   shown on the right color-coded for each fiber, is obtained by evaluating the
+   kernel density estimator along the fibers. One spurious fiber is present which
+   is isolated and badly aligned with the other fibers, and can be identified 
+   by a low LFBC value in the region where it deviates from the bundle. 
+   Figure adapted from [Portegies2015_PLoSOne]_.
+
+Here we implement FBC measures based on kernel density estimation in the 
+non-flat 5D position-orientation domain. First we compute the kernel density 
+estimator induced by the full lifted output of the tractography. Then, the 
```

Comment:
I've added some extra details here. Lifted means that it's defined in the space of positions and orientations.